<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogplanification</class>
 <widget class="QDialog" name="Dialogplanification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{border-radius: 8px; 
border: 2px solid gray;

padding: 0 8px;
background: white; }



QPushButton
{
color: gray;
font-weight: bold;
background-color: black;
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: white;
color:black;
 text-decoration: underline; }</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>                       Match Amical                      </string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>431</height>
      </rect>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Ajouter  </string>
      </attribute>
      <widget class="QLabel" name="label_a1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>581</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../green-button-png-6.png</normaloff>../green-button-png-6.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_5">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>130</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_13">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>270</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>240</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>debut</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lieu">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>190</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_12">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Afficher   </string>
      </attribute>
      <widget class="QLabel" name="label_a2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>581</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableView" name="tabmatch">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>521</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>280</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer pdf</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../small-button-clip-art-clkerm-vector-clip-art-37.png</normaloff>../small-button-clip-art-clkerm-vector-clip-art-37.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri desc</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri asc</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Modifier /Supprimer/Recherche</string>
      </attribute>
      <widget class="QLabel" name="label_a3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>621</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>341</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Lieu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Recherchid">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modlieu">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modid">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>372</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_7">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>350</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../green-button-png-6.png</normaloff>../green-button-png-6.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>320</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Heure Debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>360</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Heure Fin</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="modhdebut">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>320</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="modhfin">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>360</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="AffichRecherche">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>541</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="moddate">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>370</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_4">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../rectangle-button-icon-png-6.png</normaloff>../rectangle-button-icon-png-6.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>                          Tournois                         </string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>-10</y>
       <width>531</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>431</height>
      </rect>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Ajouter  </string>
      </attribute>
      <widget class="QLabel" name="label_a1_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditid">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lieu_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>240</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>300</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>170</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nombre d equipe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>date fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>date debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>240</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_5">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../green-button-png-6.png</normaloff>../green-button-png-6.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Afficher   </string>
      </attribute>
      <widget class="QLabel" name="label_lum2_4">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-10</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_a2_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>270</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer pdf</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../small-button-clip-art-clkerm-vector-clip-art-37.png</normaloff>../small-button-clip-art-clkerm-vector-clip-art-37.png</iconset>
       </property>
      </widget>
      <widget class="QTableView" name="tabmatch2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>551</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>270</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri desc</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri asc</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_15">
      <attribute name="title">
       <string>Modifier/Supprimer/Rechcerche   </string>
      </attribute>
      <widget class="QLabel" name="label_lum3_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>571</width>
         <height>301</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_a3_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>601</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>320</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../rectangle-button-icon-png-6.png</normaloff>../rectangle-button-icon-png-6.png</iconset>
       </property>
      </widget>
      <widget class="QLineEdit" name="modlieu_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>340</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>350</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date Fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>341</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Lieu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="modid_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>310</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_9">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../green-button-png-6.png</normaloff>../green-button-png-6.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Recherchid_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_44">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>date Debut</string>
       </property>
      </widget>
      <widget class="QTableView" name="AffichRecherche_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>541</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_45">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>380</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>nombre d'equipe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="moddd">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>310</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="moddf">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>350</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="modnbr">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>380</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
